src/core_modules/capture-core/components/Pages/New/RegistrationDataEntry/RegistrationDataEntry.component.js

```diff
@@ -1,13 +1,14 @@
 // @flow
-import React from 'react';
-import { Paper, withStyles } from '@material-ui/core';
+import React, { type ComponentType } from 'react';
+import { Grid, Paper, withStyles } from '@material-ui/core';
 import type { Props } from './RegistrationDataEntry.types';
 import { EnrollmentRegistrationEntry, TeiRegistrationEntry, SingleEventRegistrationEntry } from '../../../DataEntries';
 import { scopeTypes } from '../../../../metaData';
 import { useScopeInfo } from '../../../../hooks/useScopeInfo';
 import { useRegistrationFormInfoForSelectedScope } from '../../../DataEntries/common/useRegistrationFormInfoForSelectedScope';
 import { useScopeTitleText } from '../../../../hooks/useScopeTitleText';
 import { TrackedEntityTypeSelector } from '../../../TrackedEntityTypeSelector';
+import GeneralOutput from '../../NewRelationship/RegisterTei/GeneralOutput/GeneralOutput.container';
```

Comment:
These old files should be moved/restructured, i.e. GeneralOutput and all its underlying files. Someone else started here and I contributed to the mess later. Could you make a new issue and clean this up? One suggestion is to move the files to the DataEntries folder (but I'm open for anything).

Reply:
https://jira.dhis2.org/browse/TECH-524 added this to the sprint